### ISSUE TITLE
[BUG] Properties dialog of shape does not retain settings

### ISSUE THREAD

**Issue by Welsyntoffie** [frangoteam/FUXA#1074]
Properties dialog of shape does not retain settings.

**To Reproduce**
1. Insert shape
2. Select properties
3. Change Min Max value to Boolean (True[1-1]) - Click OK
4. No go to properties again and the previous setting reverts. Clicking OK saves the reverted settings.

**Expected behavior**
To retain the setting for Range or Boolean

**Screenshots**

**Environment**
 - FUXA version: 1.0.0 (Home assistant AddOn)
 - Node.js version: 
 - npm version:
 - Platform/OS: Home assistant OS
 - Browser: Firefox, Chrome and Brave

**Comment by MatthewReed303** [comment]
Hi @Welsyntoffie a lot of fixes have been done since v1.0.0, try the latest under releases, try via docker. The latest is now v1.1.17

**Comment by Welsyntoffie** [comment]
Was using the home assistant addon, most likely the addon version I checked. I did how ever setup a docker container for the latest version. According to about version is 1.1.18-1415.
![check0](https://github.com/frangoteam/FUXA/assets/47089904/ed4469aa-8020-4aa4-a9d6-8cd12349d3d3)
This is what I select and save then ok.

![check1](https://github.com/frangoteam/FUXA/assets/47089904/b4907592-9c28-445a-93cb-bbdf0b97375e)
This is what I get when I want to edit again.
Which is fine, because it still works as expected, but was really confusing in the beginning. I have a sensor that is True/False that did not work. I thought this was the issue. Turns out I had to convert the True/False to 1/0 via a template.
